### PR TITLE
website: initial draft of release notes framework

### PR DESCRIPTION
**Pull request by mikemorris**
We won't merge this until when we publish 1.9.0 GA, but this should be the framework where we can start writing up release notes.

### CODE REVIEW

File: website/pages/docs/release-notes/1-9-0.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# Consul 1.9.0
+
+## Release Highlights
+
+## What's Changed
+
+- Consul 1.9 is built with Go 1.15, which has dropped support for 32-bit binaries for Darwin, so we will no longer be issuing darwin_386 builds for Consul 1.9.x or newer.
+
+For more detailed information, please refer to the [changelog](https://github.com/hashicorp/consul/blob/master/CHANGELOG.md#190-beta2-november-07-2020).
```
Comment (mikemorris) [inline website/pages/docs/release-notes/1-9-0.mdx:15]: TODO: Update this link to GA changelog entry.

File: website/pages/docs/release-notes/1-9-0.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+layout: docs
+page_title: Common Error Messages
+sidebar_title: Common Error Messages
```
Comment (mikemorris) [inline website/pages/docs/release-notes/1-9-0.mdx:4]:
```suggestion
page_title: 1.9.0
sidebar_title: 1.9.0
```

**Comment by mikemorris** [comment]
@blake The 1.9.0 changelog is finalized if you want to pull content from the `UNRELEASED` section of https://github.com/hashicorp/consul/blob/52e5a2fb3245d1c2cf17046797d6e286050ad82e/CHANGELOG.md

File: website/content/docs/release-notes/1-9-0.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+layout: docs
+page_title: 1.9.0
+sidebar_title: 1.9.0
+---
+
+# Consul 1.9.0
+
+## Release Highlights
+
+- *Application-Aware Intentions:* A new set of capabilities that extends Consul’s intentions model to support Layer 7 constructs and provide the ability for operators to construct policies which, in addition to service identity, evaluate application-layer information such as HTTP Path, Headers, or Method when authorizing HTTP-based (HTTP/1.1, HTTP/2, gRPC) service-to-service communication.
```
Comment (blake) [inline website/content/docs/release-notes/1-9-0.mdx:11]:
```suggestion
- *Application-Aware Intentions:* A new set of capabilities that extends Consul’s intentions model to support Layer 7 request information. Intentions now provide the ability for operators to construct policies which evaluate application-layer information such as HTTP Path, Headers, or Method –in addition to service identity – when authorizing HTTP-based (HTTP/1.1, HTTP/2, gRPC) service-to-service communication.
```

File: website/content/docs/release-notes/1-9-0.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- *Custom Resources for Kubernetes:* In this release, we are providing a Kubernetes-first experience through CRDs to allow practitioners to easily configure Consul via Kubernetes-style objects. Specifically, we are providing the capability to manage Consul service mesh configuration for services via CRDs.
+
+- *Deploy Consul in OpenShift:* Enable installing Consul via Helm chart in OpenShift. We’ve now ensured that Consul Kubernetes runs on OpenShift securely by ensuring that Consul runs as non-root and also provided a set of SecurityContextConstraints to deploy Consul securely.
+  - Installing Consul on OpenShift should now be as simple as running a Helm install with the `global.openshift.enabled` set to `true`.
+
+- *Active Health Checks for Consul on Kubernetes:* IThis release will provide Consul service mesh integration with Kubernetes Readiness probes. This ability to natively detect health status from Kubernetes via Readiness probe is then used for directing service mesh traffic.
```
Comment (blake) [inline website/content/docs/release-notes/1-9-0.mdx:22]:
```suggestion
- *Active Health Checks for Consul on Kubernetes:* Consul service mesh now integrates with Kubernetes Readiness probes. This provides the ability to natively detect health status from Kubernetes via Readiness probe, and is then used for directing service mesh traffic.
```

File: website/content/docs/release-notes/1-9-0.mdx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+- *Streaming:* In this release, we’ve introduced streaming, a major architectural enhancement in how update notifications for blocking queries are delivered within the cluster. Streaming results in very significant reduction of CPU and network bandwidth usage on Consul servers in large-scale deployments. Streaming is particularly helpful in scaling blocking queries in Consul clusters that have rapid changes in service state.
```
Comment (blake) [inline website/content/docs/release-notes/1-9-0.mdx:24]:
```suggestion
- *Streaming:* This feature introduces a major architectural enhancement in how update notifications for blocking queries are delivered within the cluster. Streaming results in very significant reduction of CPU and network bandwidth usage on Consul servers in large-scale deployments. Streaming is particularly helpful in scaling blocking queries in Consul clusters that have rapid changes in service state.
```

File: website/content/docs/release-notes/1-9-0.mdx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  - Streaming is now available for the service health HTTP endpoint. It can be enabled through configuration flags and adding the `cached` URL parameter. In subsequent releases, we will continue to enable streaming in more endpoints.
```
Comment (blake) [inline website/content/docs/release-notes/1-9-0.mdx:25]:
```suggestion
  - Streaming is now available for the service health HTTP endpoint, and can be enabled through the [`use_streaming_backend`](https://www.consul.io/docs/agent/options#use_streaming_backend) client configuration option, and [`rpc.enable_streaming`](https://www.consul.io/docs/agent/options#rpc_enable_streaming) option on the servers. We will continue to enable streaming in more endpoints in subsequent releases.
```

File: website/content/docs/release-notes/1-9-0.mdx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+- *Active Health Checks for Consul on Kubernetes:* IThis release will provide Consul service mesh integration with Kubernetes Readiness probes. This ability to natively detect health status from Kubernetes via Readiness probe is then used for directing service mesh traffic.
+
+- *Streaming:* In this release, we’ve introduced streaming, a major architectural enhancement in how update notifications for blocking queries are delivered within the cluster. Streaming results in very significant reduction of CPU and network bandwidth usage on Consul servers in large-scale deployments. Streaming is particularly helpful in scaling blocking queries in Consul clusters that have rapid changes in service state.
+  - Streaming is now available for the service health HTTP endpoint. It can be enabled through configuration flags and adding the `cached` URL parameter. In subsequent releases, we will continue to enable streaming in more endpoints. 
+
+## What's Changed
+
+- Consul 1.9 is built with Go 1.15, which has dropped support for 32-bit binaries for Darwin, so we will no longer be issuing darwin_386 builds for Consul 1.9.x or newer.
```
Comment (blake) [inline website/content/docs/release-notes/1-9-0.mdx:29]:
```suggestion
- Consul 1.9 is built with Go 1.15, which has dropped support for 32-bit binaries for Darwin/macOS, so we will no longer be issuing darwin_386 builds for Consul 1.9.x or newer.
```

**Review by blake — COMMENTED** [review]
Overall LGTM. I had a few suggestions to change the phrasing for a few items.

File: website/content/docs/release-notes/1-9-0.mdx [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+- *Service Mesh Visualization:* This addition provides a new topology tab to the Consul UI which displays topology diagrams and key service mesh metrics like request, error rates, and timing metrics. These new features will assist developers and operators in verifying configuration and troubleshooting issues within the service mesh. The UI also supports deep linking into your external metrics dashboards.
+
+- *Improved Metrics and Grafana Dashboard:* Consul 1.9 additionally updates several metrics to reduce cardinality in metric names in favor of the best practice of using labels instead and provides a Grafana dashboard template that exposes the most important metrics associated with the Consul datacenter for operators.
+
+- *Custom Resources for Kubernetes:* Consul now provides a Kubernetes-first experience through CRDs to allow practitioners to easily configure Consul service mesh via Kubernetes-style objects, or CRDs.
```
Comment (blake) [inline website/content/docs/release-notes/1-9-0.mdx:17]:
```suggestion
- *Custom Resources for Kubernetes:* Consul now provides a Kubernetes-first experience through CRDs to allow practitioners to easily configure Consul service mesh via Kubernetes-style objects.
```
I re-read this and realized it was repetitive.

**Review by blake — APPROVED** [review]

**Pull request merged**